controllers/scopeinstance_controller.go

```diff
@@ -143,12 +192,29 @@ func (r *ScopeInstanceReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 	}
 
 	if err := r.deleteBindings(ctx, listOption, listOptions); err != nil {
+		r.updateScopeTemplateCondition(ctx, in, metav1.Condition{
+			Type:               "Succeeded",
+			Status:             metav1.ConditionFalse,
+			ObservedGeneration: in.Generation,
+			LastTransitionTime: metav1.Now(),
+			Reason:             "RoleBindingDeleteFailure",
+			Message:            fmt.Sprintf("failed to delete RoleBindings associated with ScopeInstance `%s`: %s", in.Name, err),
```

Comment:
A few nits/suggestions:
1. There's typically no need to use "failed to" or "error" sorts of prefixes/suffixes. It's almost always obvious in context.
2. Use %q to automatically get a quoted string.
3. For errors, pretty much exclusively use `%v` for plain/not-wrapped errors and `%w` for wrapped errors. And only use `%w` if you know that something handling the error will need to use `errors.Is` or `errors.As` on it.
```suggestion
			Message:            fmt.Sprintf("delete RoleBindings for %q: %v", in.Name, err),
```

Reply:
Should be resolved in commit https://github.com/awgreene/oria-operator/pull/21/commits/6f670409b0304ae9d63cc1b7e30fab1c8b19e094

Let me know how this looks!